From 5c6bb341c847ad3d793dab45c4033b05bc9654bb Mon Sep 17 00:00:00 2001
From: Baruch Siach <baruch@tkos.co.il>
Date: Wed, 13 Dec 2017 17:52:56 +0200
Subject: [PATCH] mcbin: save environment to the SD card

The Macchiatobin SD card is on mmc device 1.

The SD card does not support hardware partitions, so leave this
parameter at its default value.

Must set CONFIG_MVEBU_MMC_BOOT=y in .config for this to work.
---
 include/configs/mvebu_armada-8k.h     | 2 +-
 include/configs/mvebu_armada-common.h | 1 -
 2 files changed, 1 insertion(+), 2 deletions(-)

diff --git a/include/configs/mvebu_armada-8k.h b/include/configs/mvebu_armada-8k.h
index 87bfcbd0fdda..ce7a9e4107fd 100644
--- a/include/configs/mvebu_armada-8k.h
+++ b/include/configs/mvebu_armada-8k.h
@@ -29,7 +29,7 @@
 #define CONFIG_KIRKWOOD_SPI
 
 /* Environment in MMC */
-#define CONFIG_SYS_MMC_ENV_DEV		0
+#define CONFIG_SYS_MMC_ENV_DEV		1
 
 #define CONFIG_SYS_MAX_NAND_DEVICE	1
 #define CONFIG_SYS_NAND_MAX_CHIPS	1
diff --git a/include/configs/mvebu_armada-common.h b/include/configs/mvebu_armada-common.h
index c406a7aa1b0b..f085a49f9ddf 100644
--- a/include/configs/mvebu_armada-common.h
+++ b/include/configs/mvebu_armada-common.h
@@ -128,7 +128,6 @@
 /* Environment in MMC */
 #elif defined(CONFIG_MVEBU_MMC_BOOT)
 #define CONFIG_ENV_IS_IN_MMC
-#define CONFIG_SYS_MMC_ENV_PART		1 /* 0 - DATA, 1 - BOOT0, 2 - BOOT1 */
 /* Environment in SPI NAND flash */
 #elif defined(CONFIG_MVEBU_SPINAND_BOOT)
 #define CONFIG_ENV_IS_IN_SPI_NAND
-- 
2.15.1

